src/components/contextual/pages/vebal/MyVeBAL/components/MyVeBalCards.vue

```diff
@@ -192,13 +217,42 @@ const cards = computed(() => {
 
 <style scoped>
 .label {
-  @apply text-sm text-gray-500 font-medium mb-2;
+  @apply text-sm mb-2;
 }
 .value {
-  @apply text-xl font-medium truncate flex items-center justify-between mb-1;
+  @apply text-xl font-medium flex flex-wrap items-center justify-between mb-0.5;
 }
 
 .secondary-value {
-  @apply text-gray-400 dark:text-gray-500;
+  @apply text-sm text-gray-500 dark:text-gray-400;
+}
+
+.plus-circle:hover,
+.plus-circle:focus,
+.minus-circle:hover,
+.minus-circle:focus {
+  transform: scale(1.25);
+}
+
+.plus-circle:hover /deep/ svg.feather-plus-circle,
+.plus-circle:focus /deep/ svg.feather-plus-circle {
```

Comment:
Please don't use /deep/ it causes deprecation warnings in our build output.
```suggestion
.plus-circle:hover :deep(svg.feather-plus-circle),
.plus-circle:focus :deep(svg.feather-plus-circle) {
```